modules/ecs_fargate/datadog.tf

```
@@ -3,11 +3,22 @@
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
 # Copyright 2025-present Datadog, Inc.
 
+# Version and Install Info
 locals {
   # Datadog ECS task tags
+  version           = "0.0.0"
+  terraform_version = "1.5.0"
+
+  install_info_tool              = "terraform"
+  install_info_tool_version      = local.version
+  install_info_installer_version = local.terraform_version
```

I feel it is the opposite base on what I see in the helm chart. 

```suggestion
  install_info_tool_version      = local.terraform_version
  install_info_installer_version = local.version
```

please check my other comment in https://github.com/DataDog/datadog-cdk-constructs/pull/419/files#r2057741686